### PR TITLE
Alt Chat & Fullscreen Fixes

### PR DESCRIPTION
**Pull request by quardbreak**
I belive I didn't broke anything. Should fix some issues.
Also removes automatical OnResize from fullscreen and alt chat.
Adds fit viewport verb. Works funky with extended client view but work with default value.

### CODE REVIEW

**Comment by Lohikar** [comment]
What exactly is this doing to chat?

**Comment by quardbreak** [comment]
Changes some _alt's macro references to _alt type so _alt buttons will only change _alt ones, not original one, I believe
I should re-test it twice

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -533,4 +534,49 @@ client/verb/character_setup()
 		winset(src, "mainwindow", "menu=menu;statusbar=true")
 		winset(src, "mainwindow.mainvsplit", "pos=3x0")
 	winset(src, "mainwindow", "is-maximized=true")
-	OnResize()
```
Comment (MistakeNot4892) [inline code/modules/client/client_procs.dm:536]: Why remove OnResize()?

Comment (quardbreak) [reply code/modules/client/client_procs.dm:536]: I don't really see a reason to have it here :?

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**